src/commands.c

```diff
@@ -222,7 +223,6 @@ unsigned int CommandSpecGetAggregateFlags(RaftRedisCommandArray *array, unsigned
         const char * cmd_str = RedisModule_StringPtrLen(cmd, &cmd_len);
 
         if (!strncasecmp("asking", cmd_str, cmd_len)) {
-            /* TODO: when asking <magic> support is added, need to adjust to argv[2] */
```

Comment:
unsure why this comment was removed.

Reply:
Actually the whole `asking` section was removed in #347  - ASKING is a single command, with no args.